frontera/contrib/backends/remote/messagebus.py

```diff
@@ -78,3 +90,20 @@ def queue(self):
     @property
     def states(self):
         return None
+
+
+def aggregate_per_host(requests):
+    per_host = dict()
+    for link in requests:
+        if b'fingerprint' not in link.meta[b'domain']:
+            continue
+        host_fprint = link.meta[b'domain'][b'fingerprint']
```

Comment:
I think we should rely on DomainMiddleware always. In fact it should be hardcoded, because without it too many things will break.